packages/storage-documents/src/privacy-scan-types/privacy-scan-configuration.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ItemType, StorageDocument } from '..';
+
+// Data type stored by WCP
+export interface PrivacyScanConfiguration extends StorageDocument {
```

Comment:
do we need to have unique configuration per each scan? Will be sufficient to have privacy scan configuration defined in a code as a json file or similar?

Reply:
I discussed with Udaya, we can use a common config for the MVP but may eventually need to add options for sites with custom banners. I'll remove this file for now.